batch/2.0/_index.md

```diff
@@ -36,19 +36,20 @@ Please reference that ballot for the official results.
 
 ## Release Review
 
-The Specification Committee Ballot concluded successfully on 2020-mm-dd with the following results.
+The Specification Committee Ballot concluded successfully on 2020-08-27 with the following results.
 
 | Representative                                 | Representative for: | Vote |
 |------------------------------------------------|---------------------|------|
-| Kenji Kazumura, Michael DeNicola               | Fujitsu             |      |
-| Dan Bandera, Kevin Sutter                      | IBM                 |      |
-| Bill Shannon, Ed Bratt                         | Oracle              |      |
-| Mark Wareham, Steve Millidge                   | Payara              |      |
-| Scott Stark, Mark Little                       | Red Hat             |      |
-| David Blevins, Cesar Hernandez                 | Tomitribe           |      |
-| Ivar Grimstad                                  | EE4J PMC            |      |
-| Alex Theedom                                   | Participant Members |      |
-| Werner Keil                                    | Committer Members   |      |
-|                                                | Total               |      |
-
-The ballot was run in the [jakarta.ee-spec mailing list]()
+| Kenji Kazumura                                 | Fujitsu             |  1   |
```

Comment:
According to @ivargrimstad's comment on my PR, we should be explicit with a +1 (vs -1 or 0) in this column.  So, if you could update this column, it would be consistent with Jakarta EE 8.

Reply:
Done